docs/guides/create-composable-components.stories.mdx

```diff
@@ -0,0 +1,187 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Guides/Creat Composable Components" />
```

Comment:
```suggestion
<Meta title="Guides/Create Composable Components" />
```

Reply:
typo aside still thinking through if there's another name for the section I might suggest

Reply:
after reading most of this, I wonder if we're combining two related but fairly separate topics

it's got a mix of how to choose the way to modify an existing component/what to think about when making a new component that you want to offer flexibility/composability on and then also it's touching on how/when to customize Atlantis components

I feel like those are different scenarios that would benefit from having documents specifically talking about that topic rather than trying to find which parts are relevant to the problem I'm interested in